webpack/assets/javascripts/react_app/components/charts/Chart.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Component from '../common/BindingComponent';
+
+export default class extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    componentDidMount() {
+        this.props.drawChart();
+    }
+
+    render() {
+        this.props.drawChart();
```

Comment:
question: do you actually need to call the function on render? isnt that what did mount for?